### ISSUE TITLE
Almost all exceptions during parsing are caught & ignored

### ISSUE THREAD

**Issue by talos** [rcackerman/parole-hearing-data#23]
Ideally, we shouldn't use a general `except`, as it could hide errors that we need to fix. What exception is raised when there is a missing detail page or no results (https://github.com/rcackerman/parole-hearing-data/blob/master/scrape.py#L99 and https://github.com/rcackerman/parole-hearing-data/blob/master/scrape.py#L142)?

**Comment by rcackerman** [comment]
Sorry, I'm not sure I understand what you're trying to get from this issue. Are you asking why we put in the except clauses?

**Comment by talos** [comment]
I submitted this issue before I delved deeper into the code and realized that the `except`s were catching `AttributeError`s from when a method was called on a `None` from BeautifulSoup's `find`... I was asking what the `except` clauses were actually catching.  However, I think if we make sure that elements are not `None` we don't need them anyway.

tl;dr: I answered my own question.  Closing. :)
